bin/storm.py

```diff
@@ -241,7 +240,6 @@ def jar(jarfile, klass, *args):
             extrajars=[tmpjar, USER_CONF_DIR, STORM_BIN_DIR],
             args=args,
             daemon=False,
```

Comment:
This was supposed to make it transparent to the end user.

Why can't we return the exit code save it and then return it after the delete?